### PR TITLE
edited: README - stacks, startup

### PR DESCRIPTION
**Pull request by julycrystal**
Edited readme for better understanding of project.
Please have a look at.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,42 @@
-# Hydrogen template: Hello World
+# Shopify Storefront With Fancy Styles
```
Comment (oddcommits) [inline README.md:1]: ?

**Review by oddcommits — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (julycrystal) [reply README.md]: Fixed those issues.

File: README.md [hunk 2]
```diff
@@ -1,42 +1,42 @@
-# Hydrogen template: Hello World
+# Shopify Storefront With Custom Styles
 
-Hydrogen is Shopify’s stack for headless commerce. Hydrogen is designed to dovetail with [Remix](https://remix.run/), Shopify’s full stack web framework. This template contains a **minimal setup** of components, queries and tooling to get started with Hydrogen.
+## Stacks used
```
Comment (oddcommits) [inline README.md:3]: not `Stacks` but `Stack`. or can be `Technologies`

**Review by oddcommits — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (julycrystal) [reply README.md]: Fixed to `Technologies`

**Review by oddcommits — APPROVED** [review]

**Pull request merged**